doc/autoscale/experiment/mnist/train.py

```diff
@@ -0,0 +1,160 @@
+from PIL import Image
+import numpy as np
+import paddle.v2 as paddle
+import paddle.v2.dataset.common as common
+import os
+import sys
+import glob
+import pickle
+
+
+DC = os.getenv("PADDLE_CLOUD_CURRENT_DATACENTER")
+common.DATA_HOME = "/pfs/%s/public/idl/users/dl/paddlecloud/public/dataset" % DC
```

Comment:
The especial  path  is only used for the internal CPU cluster, I will changed this one to a general path before merged.

Reply:
Thanks!